src/utils/Processing.ts

```diff
@@ -85,17 +85,21 @@ export function loadExternalDependencies(
 }
 
 export function getIGDependencies(inDir: string): string[] {
-  const igFiles = getFilesRecursive(inDir).filter(file =>
-    file.startsWith(inDir + '/ImplementationGuide')
-  );
+  const inputFiles = getFilesRecursive(inDir);
```

Comment:
I think it would be best here to filter out non-JSON. So something like:
```suggestion
  const inputFiles = getFilesRecursive(inDir).filter(file => file.endsWith('.json'));
```